src/singletons/Settings.hpp

```diff
@@ -99,6 +101,9 @@ class Settings : public ABSettings, public ConcurrentSettings
     BoolSetting showTabLive = {"/appearance/showTabLiveButton", true};
     BoolSetting hidePreferencesButton = {"/appearance/hidePreferencesButton",
                                          false};
+    BoolSetting enableTrayIcon = {"/appearance/enableTrayIcon", false};
+    EnumSetting<TrayIconTrigger> trayIconTrigger = {
+        "/appearance/trayIconTrigger", TrayIconTrigger::OnMinimize};
```

Comment:
In my humble opinion, this could be enabled by deafult and use `TrayIconTrigger::OnClose` as a default tray icon trigger